### ISSUE TITLE
Incorrect comments on the fetched schema

### ISSUE THREAD

**Issue by pranaygp** [graphql-cli/graphql-cli#199]
The schema that's being retrieved when doing `graphql get-schema ...` generates a schema like this:

```graphql
...
"""
Comment explaining the type Asset
"""
type Asset {
  """Asset symbol"""
  symbol: String!
...
```

The problem here is `"""` isn't the right way to do comments. Comments are simply a line that beings with `#` 

The above should instead be written as:

```graphql
...
# Comment explaining the type Asset
type Asset {
  # Asset symbol
  symbol: String!
...
```

In fact, when running the `relay-compiler` on the generated schema, I get the following error:

```
ERROR:
Error loading schema. Expected the schema to be a .graphql or a .json
file, describing your GraphQL server's API. Error detail:

GraphQLError: Syntax Error GraphQL request (8:1) Unexpected String

7:
8: """
   ^
9: Comment explaining the type Asset
```

After manually changing all comments in the schema, the compiler works correctly.

P.S. Also notice how github syntax highlights the graphql snippets differently based on the comment

**Comment by kbrandwijk** [comment]
As of graphql 0.12, the syntax for Type and Field descriptions has changed to `""" Description """`. Not all tools (including the Github language parser) are updated for this yet however.

**Comment by pranaygp** [comment]
@kbrandwijk I only quickly searched through this doc for `"""` and found no hits, and documentation, examples and the comment example seems to use `#`: http://facebook.github.io/graphql/October2016/#sec-Comments

That's apparently the latest GraphQL spec. I'm just struggling to use this tool with the `relay-compiler` because of this discrepancy. I feel like I'm not the only one who's had this issue?

**Comment by pranaygp** [comment]
Actually, after some googling, you seem to be right. This is a much more recent update and it kinda sucks I can't *currently* use this tool with Relay. 

Also, since this is a duplicate of #134 and that issue agrees that `relay-compiler` needs to be updated, I'm closing this

(follow this on #2251)

**Comment by pranaygp** [comment]
UPDATE: If you're running into the same problem with `relay-compiler`, This has been fixed in relay version`1.5.0-rc.1`, which uses the latest version of `graphql ` where the fix has been implemented. 

Please update to that version (or 1.5.0 once it's released) if you want this fixed
